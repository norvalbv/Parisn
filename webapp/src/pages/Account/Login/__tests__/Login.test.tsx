import { FC } from 'react';
import renderer from 'react-test-renderer';

import LoginPage from '../';

describe('<LoginPage />', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<LoginPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
