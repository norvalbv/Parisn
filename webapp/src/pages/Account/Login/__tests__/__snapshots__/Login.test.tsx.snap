// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> should render correctly 1`] = `
<div>
  Login
</div>
`;
